src/features/defi/providers/idle/components/IdleManager/Deposit/IdleDeposit.tsx

```diff
@@ -143,7 +156,7 @@ export const IdleDeposit: React.FC<IdleDepositProps> = ({
   }
 
   return (
-    <DepositContext.Provider value={{ state, dispatch }}>
+    <DepositContext.Provider value={value}>
```

Comment:
Technically shallow comparison of props means this was previously safe, but something something not setting precedents